roles/pre_tasks/tasks/main.yml

```diff
@@ -1,10 +1,23 @@
 ---
 
 # Install absolutely necessary packages
+
+- name: Check if the absolutely necessary packages are installed
+  shell: "type {{ item }}"
+  with_items: "{{ tower_pre_tasks_packages }}"
+  register: type_packages
+  ignore_errors: yes
```

Comment:
you should add a `changed_when: false` as the command doesn't change anything.

Reply:
Updated, Agree with changes